tests/kafkatest/services/kafka/templates/kafka.properties

```diff
@@ -60,4 +60,4 @@ zookeeper.session.timeout.ms={{ zk_session_timeout }}
 replica.lag.time.max.ms={{replica_lag}}
 {% endif %}
 
-
+auto.create.topics.enable={{ auto_create_topics_enable | default("true") }}
```

Comment:
This should probably use an "if defined" check rather than an explicit default. That ensures we pick up changes to the actual default defined in KafkaConfig.